src/ui/public/kuery/functions/is.js

```diff
@@ -33,13 +33,12 @@ export function toElasticsearchQuery(node, indexPattern) {
     return { match_all: {} };
   }
   else if (fieldName === '*' && value !== '*') {
-    const userQuery = String(value);
-    const query = isDoubleQuoted(userQuery) ? userQuery : `"${userQuery}"`;
-
     return {
-      simple_query_string: {
-        query,
-        all_fields: true
+      multi_match: {
+        query: value,
+        fields: ['*'],
```

Comment:
If we rewrite this as `fields: [fieldName]`, then couldn't we remove [this piece of code](https://github.com/Bargs/kibana/blob/bb79e83ec84b79138005381ba3164a020430c14f/src/ui/public/kuery/functions/is.js#L50-L57)? Or is there some benefit to using a `match_phrase` over a `multi_match`?